hokusai/staging.yml

```diff
@@ -80,7 +78,7 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-access-log-emit-interval: '60'
     service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-name: 'artsy-elb-logs'
     service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix: 'staging-force'
-    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '59'
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
```

Comment:
I believe this is just reverting the change that didn't seem to improve things. Just confirming.

Reply:
Yeah @dleve123 suggested we go back to square-1 and set it as it was set before we started seeing errors on staging.  It's trivial to adjust this and the `KEEPALIVE_TIMEOUT` env var